physionet-django/static/bootstrap/css/bootstrap.css

```diff
@@ -4707,6 +4707,9 @@ button.close {
     max-width: 500px;
     margin: 30px auto;
   }
+  .modal-dialog.citation {
```

Comment:
It looks like you are editing the bootstrap.css, which would work but it would be overwritten if we update bootstrap. Could this go in one of the custom css files instead?

Reply:
Ahhhh interesting, did not realize! I'll move it!